### DISCUSSION TITLE
BAM adapter

### DISCUSSION THREAD

**Discussion by nataliechia** [GMOD/jbrowse-components#4742] [Q&A]
Hi, 
I am trying to align sequecing WGS. I am using Windows Jbrowse2, I don't find the appropriate adapter for uploading BAM files. Does this work on alternative version of the software, i.e. Linux? Limited coding/bioinformatics knowledge, so would be nice if you can guide me through. 

Thanks!

Kind Regards,
Natalie

**Comment by cmdcolin** [comment]
The basic way to start with JBrowse desktop looks like this

You can open your reference genome from here https://jbrowse.org/jb2/docs/quickstart_desktop/ ("Open sequence file") or just use hg19 or hg38 from the quickstart if it is human

After that, you can open up a BAM file and it's BAI index by using File->Open track
